src/app/buy-credits/buy-credits.component.html

```diff
@@ -0,0 +1,150 @@
+                    "><strong>{{ calculatedTipAmount() | exactCurrency:'£' }}</strong> donation (tip) to support Big Give
+                        <span *ngIf="calculatedTipAmount() > 0">💗</span> <!-- display heart emoji if tip > £0 -->
+                        <span *ngIf="calculatedTipAmount() === 0 && customTip() && customTipAmountField?.value">😢</span> <!-- display sad emoji if custom tip amount set to £0 -->
+                    </p>
+
+                    <p><strong>How does Big Give use tips?</strong></p>
+
+                    <p>Big Give is a registered charity (1136547). We don't charge charities for using our platform;
+                        we rely on the generosity of people who donate to leave a Voluntary Tip, which helps us run our match-funding campaigns.
+                        <br>
+                        Your tips will be going to:
+                    </p>
+                    <ul>
+                        <li>Supporting our website and team, allowing charities to double their donations to good causes.</li>
+                        <li>Providing match funds in our Anchor Match Fund. Through tips, charities will receive funding to launch their own match-funded campaign on Big Give.</li>
+                    </ul>
+                    <p>Thank you for your generosity! Your tip ensures that we can continue to support charities and causes that are important to you.</p>
+
+                    <div>
+                        <button mat-button matStepperNext>Next</button>
+                    </div>
+                </mat-step>
+                <mat-step
+                    formGroupName="giftAid"
+                    [stepControl]="giftAidGroup"
+                >
+                    <ng-template matStepLabel>Gift Aid (for tips to Big Give only)</ng-template>
+                    <p><strong>Boost your donation by 25p of Gift Aid for every £1 you donate</strong></p>
+
+                    <p>I confirm that I am a UK tax payer and I understand that if I pay less income tax and/or capital gains tax in the current tax year than the amount of
+                        Gift Aid claimed on all my donations it is my responsibility to pay the difference.</p>
+
+                    <p><strong>Note:</strong>This Gift Aid section is only for claiming Gift Aid on the tip to Big Give. If you wish to claim Gift Aid on the credit amount used for making donations, you may do so when donating
+                    to a charity's campaign.</p>
+
+                    <p><a href="">Find out more about Gift Aid </a></p>
+
+                    <mat-radio-group aria-label="Select an option" (change)="giftAidChoiceSelected($event)">
+                        <mat-radio-button value="yes" [checked]="false">Yes, I want to claim <strong>{{ giftAidAmount() | exactCurrency:'£' }}</strong> Gift Aid on my tip to Big Give, and any additional charitable donations via Big Give from the past 4 years and in the future.</mat-radio-button>
```

Comment:
Is there a reason we're not binding directly to true/false as in the donate form?

I think that binding in TS with e.g. `this.giftAidGroup.get('giftAid')?.valueChanges.subscribe(...)` is also a bit safer, because it keeps things agnostic to the cause of a change: any future programmatic updates to the form data would be the same as clicks for example.